### DISCUSSION TITLE
Signing for "GoogleSignIn-GoogleSignIn" requires a development team.

### DISCUSSION THREAD

**Discussion by mcfarljw** [capawesome-team/capacitor-firebase#271] [Q&A] [answered]
I've noticed that every time I run `npx cap run ios` or more specifically `npx cap sync ios` that my team and therefore signing certificate for `GoogleSignIn-GoogleSignIn` pod target gets unset to none (see screenshot below) which causes the build to fail. I then have to run `npx cap open ios` to manually set it again.

![Screenshot 2022-12-01 at 5 10 13 PM](https://user-images.githubusercontent.com/4167341/205171776-4c77388a-b48a-49cb-8ae0-61d9b7bea78e.png)

I'm not too savvy with the Podfile but I'm wondering if anyone else has run into this issue? Is there something I'm missing to have it persist my team for that specific pod target?

**Comment by robingenz** [comment, marked as answer]
Please have a look at this SO answer: https://stackoverflow.com/a/73765527/6731412
Thanks for pointing out. 
I will add this to the docs.

**Reply by robingenz** [reply]
Added: https://github.com/capawesome-team/capacitor-firebase/commit/899aaa95fc24a37086d9ce6613ffda7dfeb35b9d

**Reply by mcfarljw** [reply]
Perfect, thanks!
